docs/cli/kops_trust_keypair.md

```diff
@@ -0,0 +1,54 @@
+
+<!--- This file is automatically generated by make gen-cli-docs; changes should be made in the go CLI command code (under cmd/kops) -->
+
+## kops trust keypair
+
+Trust a keypair.
+
+### Synopsis
+
+Trust one or more keypairs in a keyset.
+
+ Trusting adds the certificates of the specified keypairs to trust stores. It is the reverse of thekops distrust keypair command.
```

Comment:
I'm not sure what needs to change in the go code but theres a missing space here between `the` and `kops`

Reply:
Fixed